internal/services/automation/automation_software_update_configuration_resource.go

```diff
@@ -514,29 +514,29 @@ func (m SoftwareUpdateConfigurationResource) Arguments() map[string]*pluginsdk.S
 					"start_time": {
 						Type:             pluginsdk.TypeString,
 						Optional:         true,
-						Computed:         true,
+						Computed:         !features.FourPointOhBeta(),
 						DiffSuppressFunc: suppress.RFC3339MinuteTime,
 						ValidateFunc:     validation.IsRFC3339Time,
 					},
 
 					"start_time_offset_minutes": {
 						Type:     pluginsdk.TypeFloat,
 						Optional: true,
-						Computed: true,
+						Computed: !features.FourPointOhBeta(),
 					},
 
 					"expiry_time": {
 						Type:             pluginsdk.TypeString,
 						Optional:         true,
-						Computed:         true,
+						Computed:         !features.FourPointOhBeta(),
 						DiffSuppressFunc: suppress.RFC3339MinuteTime,
 						ValidateFunc:     validation.IsRFC3339Time,
 					},
 
 					"expiry_time_offset_minutes": {
 						Type:     pluginsdk.TypeFloat,
 						Optional: true,
-						Computed: true,
+						Computed: !features.FourPointOhBeta(),
```

Comment:
Same comment here regarding properties that are times

Reply:
same as above, the api doesn't appear to set anything for this